loadtest/loadtest.go

```diff
@@ -47,9 +54,11 @@ func (lt *LoadTester) handleStatus() {
 
 // AddUser increments by one the number of concurrently active users
 func (lt *LoadTester) AddUser() error {
-	if !lt.started {
+	if !lt.hasStarted() {
```

Comment:
I would have probably opted for an atomic variable in this case since it's a simple true/false state value. We would have to use an `int32` instead of a `bool` but at least we wouldn't need the additional `mutex`. What do you think?

Reply:
That's better in fact.